db/migrate/20221115233023_create_decidim_participatory_documents_suggestions.rb

```diff
@@ -9,7 +9,9 @@ def change
       t.references :decidim_user_group, null: true, index: { name: "decidim_pd_suggestion_user_group" }
       t.string :state, default: :not_answered, index: { name: "decidim_pd_suggestion_state" }
       t.datetime :answered_at, index: { name: "decidim_pd_suggestion_answered" }
+      # rubocop:disable Rails/ThreeStateBooleanColumn
       t.boolean :answer_is_published, default: false
+      # rubocop:enable Rails/ThreeStateBooleanColumn
```

Comment:
```suggestion
      t.boolean :answer_is_published, default: false, null: false
```
Instead of the cop, let's just remove the null no?